aerospike-storage-backend/src/main/java/com/playtika/janusgraph/aerospike/operations/ReadOperations.java

```diff
@@ -28,13 +34,42 @@
     private final AerospikeOperations aerospikeOperations;
 
     private final WritePolicy getPolicy;
+    private final int bathReadThreshold;
 
-    public ReadOperations(AerospikeOperations aerospikeOperations) {
+    public ReadOperations(AerospikeOperations aerospikeOperations, int bathReadThreshold) {
         this.aerospikeOperations = aerospikeOperations;
         this.getPolicy = aerospikeOperations.getAerospikePolicyProvider().writePolicy();
+        this.bathReadThreshold = bathReadThreshold;
     }
 
     public Map<StaticBuffer,EntryList> getSlice(String storeName, List<StaticBuffer> keys, SliceQuery query, StoreTransaction txh) throws BackendException {
+        if(keys.size() == 1){
+            StaticBuffer key = keys.get(0);
```

Comment:
due to abstraction level better to extract get single element logic to separate method